packages/perseus/src/widgets/input-number/input-number.test.ts

```diff
@@ -335,18 +335,22 @@ describe("focus state", () => {
         const gotFocus = await act(() => renderer.focus());
 
         // Assert
+        expect(screen.getByRole("textbox")).toHaveFocus();
         expect(gotFocus).toBe(true);
     });
 
     it("supports blurring", async () => {
         //  Arrange
         const {renderer} = renderQuestion(question);
+        await act(() => renderer.focus());
```

Comment:
As we move forward with new versions of testing-library and React, it becomes more picky about DOM updates not done within `act()`. These calls definitely mutate the DOM so they need to be wrapped!